test/net/sourceforge/kolmafia/KoLAdventureValidationTest.java

```diff
@@ -918,6 +919,30 @@ public void canVisitBombedHippyCampAfterWossname() {
         assertTrue(zones.get(AdventurePool.BOMBED_HIPPY_CAMP).canAdventure());
       }
     }
+
+    @Test
+    public void changesIntoWarFratOutfitIfNecessary() {
+      setupFakeClient();
+
+      var cleanups =
+          new Cleanups(
+              withEquippableItem("beer helmet"),
```

Comment:
I didn't know about withEquippableItem. That simplifies a lot of my tests!

Reply:
Amazing! I think it was originally made by ryo